modules/tinymce/tools/tasks/bundle.js

```diff
@@ -29,13 +29,16 @@ module.exports = function (grunt) {
     options.themeFileName = options.themeFileName || "theme.min.js";
     options.pluginsDir = options.pluginsDir || "plugins";
     options.pluginFileName = options.pluginFileName || "plugin.min.js";
+    options.iconsDir = options.iconsDir || "icons";
+    options.iconsFileName = options.iconsFileName || "icons.min.js";
     options.outputPath = options.outputPath || "full.min.js";
 
     themes = grunt.option("themes");
     plugins = grunt.option("plugins");
+    icons = grunt.option("icons") || 'default';
 
-    if (!themes && !plugins) {
-      grunt.log.writeln("Use: grunt bundle --themes <comma separated list of themes> --plugins <comma separated list of plugins>");
+    if (!themes && !plugins && !icons) {
```

Comment:
icons will never be falsy due to the `|| 'default'` 🤔

Reply:
Oops, I added the default later and forgot to remove this. Thanks for the catch!

Reply:
Fixed